src/src/AdminBundle/Entity/EventType.php

```diff
@@ -33,7 +33,8 @@ class EventType extends BaseEntity
      * @var string
      *
      * @ORM\Column(name="slug", type="string", length=50)
-     * @Assert\NotBlank()
+     * @Assert\Regex("/^[a-zA-Z0-9\-]+$/", message="Slug can contain only alphanumerical characters and a hyphen")
+     * @Assert\Regex("/^admin$/", match=false, message="Slug can not be admin")
```

Comment:
osetrili sme niekde aj to, ze slug sa nebude dat zmenit? Ci dohodli sme sa ze sa bude dat menit?

Reply:
nie, to sme zatial neriesili, mozes to pridat niekam do issues. Osobne si vsak myslim, ze by sa mal dat menit a user by si mal byt vedomi toho, ake problemy to moze sposobit.

Reply:
Ked to moze spravit velke problemy, tak by nemal byt menitelny. Ved aj tak si vsade pamatame idecka a nie slugy. Takze podla mna sa nemozu stat velke problemy...

Reply:
rozosles, 5000ludom email:

```
Ahoj {meno},
zaregistruj sa na http://events.sk/super-dupper-event
```

Uvedomis si vsak, ze to malo byt `super-duper-event` tak slug premenujes. 5000ludi teraz dostatlo email s linkom ktory nikam nesmeruje.

Reply:
Ah, chapem. hmm, mozno nebudeme davat slug do urliek, alebo osetrime aby slug nebol menitelny.

Reply:
Actually, cely jediny vyznam slugu je kvoli `url`, takze ano, nemal by byt menitelny.

Reply:
Oki, takze dam tie zvysme dve veci to zvlast taskov.